docs/bridge/docs/01-About/02-Brand-Assets.md

```
@@ -0,0 +1,21 @@
+# Brand Assets
+
+### [synapse-brand-assets.zip](/brand-assets/synapse-brand-assets.zip)
```

Markdownlint

Expected: h2; Actual: h3
Heading levels should only increment by one level at a time

(MD001, heading-increment)